### DISCUSSION TITLE
How to disable gitlabci from detected dependencies in dashboard

### DISCUSSION THREAD

**Discussion by glensc** [renovatebot/renovate#22286] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.80.1

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

i have this in package rules:

```json
  "packageRules": [
    {
      "matchManagers": [
        "gitlabci",
        "gitlabci-include",
      ],
      "enabled": false,
    },
  ],
```

based on info from here:
- https://github.com/renovatebot/renovate/discussions/22027

### What you tried so far.

I tried to disable same thing, but different reason:
- https://gist.github.com/glensc/fc15ab8a8df5db16ed71d28b7ca5a39a 

### Relevant debug logs

<details><summary>Logs</summary>

```

DEBUG: migrated config (repository=glen/playground)
       "config": {
         "$schema": "https://docs.renovatebot.com/renovate-schema.json",
         "extends": [
           "..redacted..",
           ":assignAndReview(glen)"
         ],
         "packageRules": [
           {"matchManagers": ["gitlabci", "gitlabci-include"], "enabled": false}
         ]
       }

...
DEBUG: Using file match: \.gitlab-ci\.yml$ for manager gitlabci (repository=glen/playground)
DEBUG: Using file match: \.gitlab-ci\.yml$ for manager gitlabci-include (repository=glen/playground)
...

DEBUG: gitlabci is disabled (repository=glen/playground)
DEBUG: Matched 1 file(s) for manager gitlabci-include: .gitlab-ci.yml (repository=glen/playground)
DEBUG: Matched 1 file(s) for manager npm: package.json (repository=glen/playground)

...
DEBUG: manager extract durations (ms) (repository=glen/playground)
       "managers": {"gitlabci": 1, "gitlabci-include": 8, "npm": 199}
```

</details>

**Comment by viceice** [comment]
your sample is not valid JSON

**Reply by glensc** [reply]
yes it is valid. see the logs. it's json5 file. but edited for clarity.

**Comment by rarkins** [comment]
I think what's happening is that:
- Renovate scans all managers, including these gitlab ones
- The package rule is then applied, which disables all the dependencies - no lookup done
- These package files + dependencies are still there and listed in dependency dashboard issue

This might work to prevent them being looked up:
```
{
  "gitlabci": {
    "enabled": false
  },
  "gitlabci-includes": {
    "enabled": false
  }
}
```
